JavaScript/packages/recognizers-date-time/src/dateTime/english/baseConfiguration.ts

```diff
@@ -41,7 +41,7 @@ export class EnglishDateTimeUtilityConfiguration implements IDateTimeUtilityConf
 }
 
 export class EnglishCommonDateTimeParserConfiguration extends BaseDateParserConfiguration {
-    constructor() {
+    constructor(dmyDateFormat: boolean = false) {
```

Comment:
I recommend using true as the default value for English.

Reply:
English US is mdy, not dmy, so the default should be false.

Reply:
@Grey0202, please make sure English default is MDY, but all other languages are DMY. You can check consistency with .Net.